scylla-cql/src/types/deserialize/value.rs

```diff
@@ -1830,6 +1830,9 @@ pub enum BuiltinDeserializationErrorKind {
 
     /// A deserialization failure specific to a CQL UDT.
     UdtError(UdtDeserializationErrorKind),
+
+    /// Deserialization of this CQL type is not supported by the driver
+    Unuspported,
```

Comment:
:pick: Let's be consistent about dots at the end of docstrings here (and everywhere else). It matters at least for me, so perhaps for other readers too.